bool is_warm
fiducial_msgs/FiducialTransform transform